src/components/Dialog/useDialog.tsx

```diff
@@ -1,13 +1,19 @@
 import { useCallback, useMemo, useState } from 'react';
 
+export interface DialogState {
+  open: boolean;
+  onClose: () => void;
+  onExited: () => void;
+}
```

Comment:
This file should be reverted. See comment on `EditPartnershipProps`

Reply:
reverted